src/app/components/drawer/Projects/ProjectsComponent.js

```diff
@@ -324,6 +328,11 @@ const ProjectsComponent = ({
               { feedsExpanded ? <ExpandLessIcon className={styles.listChevron} /> : <ExpandMoreIcon className={styles.listChevron} /> }
               <ListItemText disableTypography className={styles.listHeaderLabel}>
                 <FormattedMessage tagName="span" id="projectsComponent.sharedFeeds" defaultMessage="Shared feeds" description="Feeds of content shared across workspaces" />
+                <Can permissions={team.permissions} permission="create Feed">
```

Comment:
@amoedoamorim can we put a tooltip on this new button, and might as well add one for "New Folder" and "New Custom List" as well?